# $NetBSD: patch-bt,v 1.1 2002/09/23 14:37:19 wennmach Exp $

Fix a problem with VOP_PUTPAGES on NetBSD-1.6. From Love <lha@netbsd.org>.

diff -u -u -w -r1.87.2.8 -r1.87.2.9
--- xfs/bsd/xfs_vnodeops-bsd.c	2002/08/02 07:46:57	1.87.2.8
+++ xfs/bsd/xfs_vnodeops-bsd.c	2002/09/22 15:25:40	1.87.2.9
@@ -974,6 +974,9 @@
 
     XFSDEB(XDEBVNOPS, ("xfs_putpages\n"));
 
+    if (t == NULL)
+	return 0;
+
     xn->flags |= XFS_DATA_DIRTY;
 
 #ifdef HAVE_STRUCT_VOP_PUTPAGES_ARGS_A_SYNC /* FreeBSD-style */

